src/copyright/agent/copyright.cc

```diff
@@ -54,7 +54,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #include "copyright.hpp"
 
 #ifndef DISABLE_JSON
```

Comment:
we are installing the lib now so "#ifndef DISABLE_JSON"  can be removed,

Reply:
Done